packages/select/src/components/query-list/queryList.tsx

```diff
@@ -231,6 +269,28 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         }
     }
 
+    private defaultMenuRenderer = (listProps: IItemListRendererProps<T>) => {
```

Comment:
Would have suggested `defaultItemListRenderer` just for consistency, but that's a mouthful isn't it?